packages/@aws-cdk/aws-iot-actions/README.md

```diff
@@ -209,3 +210,30 @@ const topicRule = new iot.TopicRule(this, 'TopicRule', {
   ],
 });
 ```
+
+## Send messages to a SQS queue
+
+The code snippet below creates an AWS IoT Rule that send messages
+to a SQS queue when it is triggered.
+
+```ts
+import * as iot from '@aws-cdk/aws-iot';
+import * as actions from '@aws-cdk/aws-iot-actions';
+import * as sqs from '@aws-cdk/aws-sqs';
+
+const queue = new sqs.Queue(this, 'MyQueue');
+const stream = new firehose.DeliveryStream(this, 'MyStream', {
+  destinations: [new destinations.S3Bucket(bucket)],
+});
```

Comment:
Why does this example contain a Firehose `DeliveryStream`? It's not used anywhere from what I can see.

Reply:
Thank you. I fixed.